### PR TITLE
Improve list bridges command output, upgrade to discord v13

### PR DESCRIPTION
**Pull request by Xyaren**

### CODE REVIEW

File: src/BotClient.ts [hunk 1]
```diff
@@ -159,10 +169,22 @@ export class Util {
         }
     }
 
-    static formatBridge(bid: number, bridge?: ResolvedBridge): string {
-        return bridge === undefined
-            ? `\`${bid}\`: INVALID`
-            : `\`${bridge.bridge_id}\`: \`${bridge.source_guild.name}#${bridge.source_channel.name}\` → \`${bridge.destination_guild.name}#${bridge.destination_channel.name}\` on condition \`${bridge.attribute}:${bridge.regex}\``;
+    static formatBridge(bid: number, bridge?: ResolvedBridge): { title: string; content: string } {
```
Comment (ogrady) [inline src/BotClient.ts:172]: Really not a fan of this style with multiple returns. Would you mind reworking this either into a ternary expression or in the style of
```javascript
const result = {
  title: title,
  content: "INVALID"
}

if (bridge !== undefined) {
  // ...
}

return result;
```
?

File: src/BotClient.ts [hunk 2]
```diff
@@ -176,18 +198,17 @@ export class Util {
...
      * @returns the TextChannel, if a channel passing the predicate was found,
-     *          or undefined if no such channel was found or g is falsey.
+     *          or null if no such channel was found or g is falsey.
      */
     static async findTextChannel(g: discord.Guild | null, phrase: string): Promise<TextChannel | NewsChannel | null> {
         let guild = await g?.fetch();
         if (guild) {
             let resolve = guild.channels.resolve(phrase) || guild.channels.cache.filter(c => c.name === phrase).first() || null;
-            if (resolve?.isText()) {
+            if (resolve?.isText() && !resolve.isThread()) {
```
Comment (ogrady) [inline src/BotClient.ts:211]: 👍 important addition

**Review by ogrady — CHANGES_REQUESTED** [review]
Really valuable contribution regarding threads, just one minor style-problem I'd prefer to see addressed before merging.

File: src/BotClient.ts (hunk 1 above)
Comment (Xyaren) [reply src/BotClient.ts:172]: fixed

**Pull request merged**